src/components/Board.tsx

```diff
@@ -0,0 +1,32 @@
+import "./Board.css";
+import Square from "./Square";
+
+interface Props {
+  winner: string;
+  squares: Array<string>;
+  onClick(index: number): void;
+  onRestart(): void;
+}
+
+function Board(props: Props) {
+  const { winner, squares, onClick, onRestart } = props;
+  return (
+    <div>
```

Comment:
Option: for `<div>` with no props, can use `<>` or `<Fragment>` instead
https://reactjs.org/docs/fragments.html

Reply:
Added: if you don't use any attribute (such key) you can go for `<>`. But if you have to, use `<Fragment>`